### PR TITLE
chore: label incoming issues with a triage label

### LINKED ISSUE #8960: chore: Label incoming issues with priority-5-triage label
**What would you like Renovate to be able to do?**

We need a way to remind ourselves to add a proper priority label to incoming issues.

I'm seeing new incoming issues which do not have a priority label.
I suspect this is because it's not longer a fully manual job to add labels, that the maintainers and triagist forget to add the priority label.

**Did you already have any implementation ideas?**

I would like to propose we add a `priority-5-triage` label to the repository.
We also adjust the issue templates to add this new label to incoming issues.
This will serve as a reminder to maintainers/triagists that they should assign a proper priority to the issue.
As an added benefit, we can search for the new triage label to find any issues that don't yet have a "real" priority.

I'm suggesting this because we now add some labels to incoming issues automatically.
This makes it easy to forget to add a `priority` label.
Maybe having a `triage` label can help with this?

I'm willing to make a PR to add the label to the issue templates, but I cannot add a new label myself, as I don't have the rights to do that. 😄 

Or maybe you guys have a better idea? 😄

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Incoming issues will be labeled `priority-5-triage`
- Update "issue labeling" docs
- Add "search URL" to find issues which are missing any priority labels

## Context:

Closes #8960.

@rarkins You will need to create the new `priority-5-label` before merging this, or GitHub will complain that it can't find the label on incoming issues. 👻 😱 

Marking this PR "draft" until we have the label present on the repository to prevent annoyances on new issues. 😉 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

**Comment by HonkingGoose** [comment]
@rarkins I do like your idea of having a search to identify issues missing a priority label, so why not have both the new triage label, and a new search. 🚀  That way we can both be happy! 😄

File: docs/development/issue-labeling.md [hunk 1]
```diff
@@ -63,14 +63,18 @@ Add the `breaking` label for Issues or PRs which contain changes that are not ba
...
 
 </details>
 
 Use these to assign a priority level to an issue.
+Incoming issues are labeled `priority-5-triage` by default, this label should be replaced with a proper priority (low/normal/important/critical).
 Make a best-effort attempt to select a proper priority.
 Nothing bad will happen if you select a "wrong" priority.
 At a high level: critical = needs immediate fix, important = to be prioritized ahead of others, normal = default priority, low = trivial issue, or impacts a very small % of the user base.
 
+Use [this search](https://github.com/renovatebot/renovate/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+-label%3Apriority-1-critical+-label%3Apriority-2-important+-label%3Apriority-3-normal+-label%3Apriority-4-low++-label%3Apriority-5-triage) to find any issues which are missing a priority label.
```
Comment (HonkingGoose) [inline docs/development/issue-labeling.md:76]: Please check my work on the search query before approving again.

Comment (rarkins) [reply docs/development/issue-labeling.md:76]: Yes, let's looks right. Shall we bulk label those as `priority-5-triage`?

Comment (HonkingGoose) [reply docs/development/issue-labeling.md:76]: Yeah let's bulk label those. I'll let you do the honors on that one. 😄

Comment (HonkingGoose) [reply docs/development/issue-labeling.md:76]: I don't think you can actually bulk label those...
I don't see a checkbox before the issue to select it from the search screen.
So it seems bulk labeling is right out.

@rarkins Want me to label the issues manually then?

Comment (HonkingGoose) [reply docs/development/issue-labeling.md:76]: Oh you probably need `maintainer` level rights to bulk label things... 😄

Comment (viceice) [reply docs/development/issue-labeling.md:76]: image
🙃

**Review by viceice — APPROVED** [review]

File: docs/development/issue-labeling.md (hunk 1 above)
Comment (HonkingGoose) [reply docs/development/issue-labeling.md:76]: Thank you @viceice for doing the bulk labeling. 😄 

I'm resolving this conversation.

**Pull request merged**